kie-drools-wb/kie-drools-wb-distribution-wars/src/main/assembly/component-kie-drools-wb-tomcat-7-common.xml

```diff
@@ -98,8 +100,6 @@
           <!-- Errai -->
           <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
           <exclude>WEB-INF/lib/errai-cdi-server-*.jar</exclude>
```

Comment:
@csadilek sorry to ask again, but I am getting lost here. Since we upgraded to Weld 2.x (CDI 1.2), should we still exclude errai-cdi-server?

I also noticed the WAR file contains errai-cdi-client. Is that correct?